DQM/RPCMonitorDigi/interface/utils.h

```diff
@@ -13,6 +13,21 @@ namespace rpcdqm{
 
   enum RPCMeFLag{OCCUPANCY = 1, CLUSTERSIZE = 2, MULTIPLICITY =3, BX=4};
 
+  struct RPCMEHelper
+  {
+    static void setNoAlphanumeric(MonitorElement* myMe)
+    {
+      // Set no-alphanumeric flag to avoid malfunctioning in multithread environment.
+      TH2* h2 = nullptr;
```

Comment:
hi @dmitrijus @jhgoh - could you add a getTH2() method to MonitorElement to avoid this complexity. Thanks

Reply:
I can simply use getTH1() since TH2 inherits from TH1.